docs/feature-store/sources-targets.md

```diff
@@ -0,0 +1,47 @@
+| [mlrun.datastore.BigQuerySource](../api/mlrun.datastore.html#mlrun.datastore.BigQuerySource)                      | Reads Google BigQuery query results as input source for a flow.                 | N      | Y     | Y      |batch|
+| mlrun.datastore.SnowFlakeSource                                                                                 | Reads Snowflake query results as input source for a flow.                         | N      | Y     | N      |batch|
+| mlrun.datastore.SQLSource                                                                                       | Reads SQL query results as input source for a flow.                               | Y      | N     | Y      |batch|
+| [mlrun.datastore.CSVSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.CSVSource)            | Reads a CSV file as input source for a flow.                                    | Y      | Y     | Y      |batch|
+| [storey.sources.DataframeSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.DataframeSource) | Reads data frame as input source for a flow.                                    | Y      | N     | N      |batch|
+| [mlrun.datastore.HttpSource](../api/mlrun.datastore.html#mlrun.datastore.HttpSource)                              | Sets the HTTP-endpoint source for the flow.                                     | Y      | N     | N      |event|
+| [mlrun.datastore.KafkaSource](../api/mlrun.datastore.html#mlrun.datastore.KafkaSource)                            | Sets the kafka source for the flow.                                             | Y      | N     | N      |event|
+| [mlrun.datastore.ParquetSource](https://storey.readthedocs.io/en/latest/api.html#storey.sources.ParquetSource)    | Reads the Parquet file/dir as the input source for a flow.                      | Y      | Y     | Y      |batch|
+| [mlrun.datastore.StreamSource](../api/mlrun.datastore.html#mlrun.datastore.StreamSource)                          | Sets the stream source for the flow. If the stream doesn’t exist it creates it. | Y      | N     | N      |event|
+
+## Targets
+| Class name                                                                                                    | Description                                                                                        | storey | spark | pandas |Online or offline|
+| --------------------------------------------------                                                            | -------------------------------------------------------                                            | ---    | ---   | ---    |---|
+| [mlrun.datastore.CSVTarget](https://storey.readthedocs.io/en/latest/api.html#storey.targets.CSVTarget)        | Writes events to a CSV file.                                                                       | Y      | Y     | Y      |offline|
```

Comment:
Same here, offline/online is an attribute of the target unrelated to whether a particular ingestion engine support it.
Suggest to add to the Description column.